karma.conf.js

```diff
@@ -72,7 +73,7 @@ module.exports = function (config) {
         // start these browsers
         // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
         // you can also add Chrome or other browsers too
-        browsers: ['PhantomJS'],
+        browsers: ['Chrome'],
```

Comment:
Ah yes, I see that now. In that case, no preference on that change. In terms of browser speed, it goes 1. PhantomJS, 2. Chrome, 3. Firefox. The slower your browser, the more like you are to not-catch flaky scenarios in local development. That's the reason I was originally asking. But I see your change on PhantomJS above, and that's good by me.